MetaMask/metamask-extension
# window open noopener

## Explanation

This is part of work on preventing [reverse tabnabbing](https://swisskyrepo.github.io/PayloadsAllTheThings/Tabnabbing/#summary) fixing [security#149](https://github.com/MetaMask/pm-security/issues/149)

## Manual Testing Steps
visit [chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/home.html#new-account/connect](chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/home.html#new-account/connect)
click anyone of the options
click "Buy Now"
open devtools in the new opened window
in console type in `opener` and press enter, it should return null



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
